backend/src/apiserver/storage/job_store.go

```diff
@@ -490,10 +490,10 @@ func (s *JobStore) UpdateJob(swf *util.ScheduledWorkflow) error {
 }
 
 // factory function for job store.
-func NewJobStore(db *DB, time util.TimeInterface) *JobStore {
+func NewJobStore(db *DB, time util.TimeInterface, pipelineStore PipelineStoreInterface) *JobStore {
```

Comment:
Could you add a comment documenting that `pipelineStore` is not needed if the pipeline store is the database?